feat: add sso (enterprise login) docs under platform (#10924)
supabase/supabase: apps/docs/pages/guides/platform/sso/sso-with-okta.mdx

contact.

![Okta dashboard: Attribute Statements configuration
screen](/docs/img/sso-okta-step-05.png)

## Step 6: Obtain IdP metadata URL [#idp-metadata-url]

Supabase needs to finalize enabling single sign-on with your Okta
application.

To do this scroll down to the *SAML Signing Certificates* section on the *Sign
On* tab of the *Supabase* application. Pick the the *SHA-2* row with an
*Active* status. Click on the *Actions* dropdown button and then on the *View
IdP Metadata*. 

This will open up the SAML 2.0 Metadata XML file in a new tab in your browser.
Copy this URL and send it to your support contact and await further
instructions. If you're not clear who to send this link to or need further
assistance, please reach out to
[support@supabase.com](mailto:support@supabase.com).

The link usually has this structure: `https://<okta-org>.okta.com/apps/<app-id>/sso/saml/metadata`

![Okta dashboard: SAML Signing Certificates, Actions button highlighted](/docs/img/sso-okta-step-06.png)

## Step 7: Wait for confirmation [#confirmation]

Once you’ve configured the Okta app as shown above, make sure you send the

> [metadata URL](idp-metadata-url) and information regarding the

Reviewer: I forgot the `#` here 😅 
```suggestion
[metadata URL](#idp-metadata-url) and information regarding the
```

import Layout from '~/layouts/DefaultGuideLayout'

export const meta = {
  id: 'sso-with-okta',

>   title: 'Setup SSO with Okta',
>   description: 'Configuration instructions for single sign-on with Okta',

Reviewer: ```suggestion
  title: 'Set Up SSO with Okta',
  description: 'Configure single sign-on with Okta.',
```

}

Supabase supports single sign-on (SSO) using Okta.

> ## Step-by-step guide
> 
> ### Step 1: Choose Create App Integration in the Applications dashboard

Reviewer: ```suggestion
## Step 1: Choose Create App Integration in the Applications dashboard [#create-app-integration]
```

Navigate to the Applications dashboard of the Okta admin console. Choose the
*Create App Integration* button from the toolbar.

![Okta dashboard: Create App Integration
button](/docs/img/sso-okta-step-01.png)

> ### Step 2: Choose SAML 2.0 in the app integration dialog

Reviewer: ```suggestion
## Step 2: Choose SAML 2.0 in the app integration dialog [#create-saml-app]
```

Supabase supports the SAML 2.0 SSO protocol. Choose it from the *Create a new
app integration* dialog.

![Okta dashboard: Create new app integration dialog](/docs/img/sso-okta-step-02.png)

> ### Step 3: Fill out General Settings

Reviewer: ```suggestion
## Step 3: Fill out General Settings [#add-general-settings]
```

The information you enter here is for visibility into your Okta applications
menu.  You can choose any values you like. `Supabase` as a name works well for
most use cases.

![Okta dashboard: Create SAML Integration
wizard](/docs/img/sso-okta-step-03.png)

> ### Step 4: Fill out SAML Settings

Reviewer: ```suggestion
## Step 4: Fill out SAML Settings [#add-saml-settings]
```

These settings let Supabase use SAML 2.0 properly with your Okta application.
Make sure you enter this information exactly as shown on in this table and
screenshot.

| Setting | Value |
| ------- | ----- |
| Single sign-on URL | `https://app.supabase.com/auth/v1/sso/saml/acs` |
| Use this for Recipient URL and Destination URL |✔️ |
| Audience URI (SP Entity ID) | `https://app.supabase.com/auth/v1/sso/saml/metadata` |
| Default RelayState | `https://app.supabase.com` |
| Name ID format | EmailAddress |
| Application username | Email |
| Update application username on | Create and update |

![Okta dashboard: Create SAML Integration
wizard, Configure SAML step](/docs/img/sso-okta-step-04.png)

> ### Step 5: Fill out Attribute Statements

Reviewer: ```suggestion
## Step 5: Fill out Attribute Statements [#add-attribute-statements]
```

Attribute Statements allow Supabase to get information about your Okta users on each login.

**A `email` to `user.email` statement is required to exist.** Other mappings
shown below are optional and configurable depending on your Okta
setup. If in doubt, replicate the same config as shown.

Please share any changes, if any, from this screen with your Supabase support
contact.

![Okta dashboard: Attribute Statements configuration
screen](/docs/img/sso-okta-step-05.png)

> ### Step 6: Obtain IdP metadata URL

Reviewer: ```suggestion
## Step 6: Obtain IdP metadata URL [#idp-metadata-url]
```

Supabase needs to finalize enabling single sign-on with your Okta
application.

To do this scroll down to the *SAML Signing Certificates* section on the *Sign
On* tab of the *Supabase* application. Pick the the *SHA-2* row with an
*Active* status. Click on the *Actions* dropdown button and then on the *View
IdP Metadata*. 

This will open up the SAML 2.0 Metadata XML file in a new tab in your browser.
Copy this URL and send it to your support contact and await further
instructions. If you're not clear who to send this link to or need further
assistance, please reach out to
[support@supabase.com](mailto:support@supabase.com).

The link usually has this structure: `https://<okta-org>.okta.com/apps/<app-id>/sso/saml/metadata`

![Okta dashboard: SAML Signing Certificates, Actions button highlighted](/docs/img/sso-okta-step-06.png)

> ### Step 7: Wait for confirmation

Reviewer: ```suggestion
## Step 7: Wait for confirmation [#confirmation]
```

Once you’ve configured the Okta app as shown above, make sure you send the

> metadata URL from [Step 6](#step-6-obtain-idp-metadata-url) and information
> regarding the attribute statements from [Step
> 5](#step-5-fill-out-attribute-statements) (if any changes are applicable) to

Reviewer: ```suggestion
[metadata URL](idp-metadata-url) and information regarding the
[attribute statements](#add-attribute-statements) (if any changes are applicable) to
```

your support contact at Supabase.

Wait for confirmation that this information has successfully been added to
Supabase. It usually takes us 1 business day to configure this information
for you.

> ### Step 8: Test single sign-on

Reviewer: ```suggestion
## Step 8: Test single sign-on [#testing]
```
